requirements/common.txt

```diff
@@ -18,7 +18,7 @@ pandas>=1.3
 plotly==5.9.0
 pprintpp==0.4.0
 psutil>=5.7.0
-pymongo>=3.11
+pymongo>=4.3.2
```

Comment:
Best to update `motor>=3.1.1` too: https://motor.readthedocs.io/en/stable/requirements.html

Reply:
whoops I thought it was